env0/resource_template.go

```diff
@@ -385,14 +391,10 @@ func templateCreatePayloadFromParameters(prefix string, d *schema.ResourceData)
 
 	isNew := d.IsNewResource()
 
-	tokenIdKey := "token_id"
```

Comment:
not required anymore. Gitlab should be explicitly configured. This is a breaking change that was discussed in Slack.
Need to add to release notes.